checks/binary_artifact.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checks
+
+import (
+	"strings"
+
+	"github.com/ossf/scorecard/checker"
+)
+
+//nolint
+func init() {
+	registerCheck("Binary-Artifact", BinaryArtifacts)
+}
+
+// BinaryArtifacts  will check the repository if it contains binary artifacts.
+func BinaryArtifacts(c *checker.CheckRequest) checker.CheckResult {
+	return CheckIfFileExists(check, c, binaryfileExists)
```

Comment:
`check` is declared here - https://github.com/ossf/scorecard/blob/main/checks/automatic_dependency_update.go#L23, we shouldn't be using that. 

On another note, could you also update `automatic_dependency_update` to not use  a generic var like `check` and instead update it to a exported var?